figures/3.figure3/scripts/figure3.r

```diff
@@ -747,7 +742,7 @@ fig3 <- (
     + sub_figure3
     + plot_layout(design = layout)
     # make bottom plot not align
-    + plot_annotation(tag_levels = list(c("A", "B", "C"))) & theme(plot.tag = element_text(size = 20))
+    + plot_annotation(tag_levels = list(c("A", "B", "C", "D", "E", "F", "G"))) & theme(plot.tag = element_text(size = 20))
```

Comment:
oh, one other comment. Please check that we reference _all_ tags in the manuscript. Do we? I can see an alternative being that it is labeled "D" and we refer to all of these unique feature overlaps at once.